packages/deploy-aws/src/seagull_stack.ts

```diff
@@ -99,20 +99,28 @@ export class SeagullStack extends Stack {
     const name = `${this.id}-${cfdName}`
     const domainName = getApiGatewayDomain(props.apiGateway.url)
     const originPath = getApiGatewayPath(props.apiGateway.url)
-    const defaultBehavior = {
-      allowedMethods: CF.CloudFrontAllowedMethods.ALL,
-      compress: true,
-      forwardedValues: { headers: ['authorization'], queryString: true },
-      isDefaultBehavior: true,
-    }
+    const defaultBehavior = this.getDefaultBehavior()
+
     const behaviors = [defaultBehavior]
```

Comment:
const behaviors = [this.getDefaultBehavior()]
as long as defaultBehavior is not needed otherwise I would write it that way